docs/external-node.md

```diff
@@ -561,3 +562,39 @@ interfaces will be added in the future.
 
 `ExternalNode` name must be unique in the `cluster` scope even though it is
 itself a Namespaced resource.
+
+## Known issues
+
+After installing `antrea-agent` on VM, `antrea-agent` fails to attach the VM's
+network interface to the OVS bridge if the VM is rebooted. On Windows VM,
+network connectivity might be lost after reboot.
+
+As a workaround to [issue #4122](https://github.com/antrea-io/antrea/issues/4122),
```

Comment:
As a workaround for [this issue](https://github.com/antrea-io/antrea/issues/4122), you can manually

Reply:
updated.